events/opium_wars_events.txt

```diff
@@ -158,9 +158,16 @@ opium_wars.1 = {
 		name = opium_wars.1.b
 		add_modifier = {
 			name = opium_wars_addiction_modifier
-			years = 40 #normal_modifier_time
+			years = 40 # very_long_modifier_time
 			is_decaying = yes
 		}
+		add_radicals = {
+			value = medium_radicals
+		}		
+		if = {
+			limit = { has_journal_entry = je_warlord_china }
+			chi_remove_1_fragile_unity = yes
+		}				
```

Comment:
We need to implement this regarding the changed approach to fragile unity...

Reply:
Do we have a problem with fragile unity and warlords?

Reply:
Fragile unity has gone from a scale from 0 - 5 to 0 - 100. And it uses a scripted value to calculate how much it has.
So these lines would do nothing.

Reply:
The script value needs to be updated

Reply:
I made it a todo